File: PRAK401-2410817310014-INDRA_SURYADILAGA.py
bil_kelipatan, tanda = map(str, input().split())
bil_kelipatan = int(bil_kelipatan)
bilangan = 1

while bilangan < 51:
    if bilangan % bil_kelipatan == 0:
        print(tanda, end=" ")
    else:
        print(bilangan, end=" ")
    bilangan += 1

File: PRAK401-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int bilangan, bil_kelipatan ;
    char tanda ;

    scanf("%d %c", &bil_kelipatan, &tanda) ;
    bilangan = 1 ;

    while(bilangan < 51) {
        if (bilangan % bil_kelipatan != 0){
            printf("%d ", bilangan) ;
        }
        else {
        printf("%c ", tanda) ;
        }
        bilangan ++;
    }
    return 0;
}